src/cli/constant.js

```diff
@@ -71,6 +71,16 @@ const categoryOrder = [
  */
 /* eslint sort-keys: "error" */
 const options = {
+  cache: {
+    default: false,
+    description: "Only format changed files",
+    type: "boolean",
+  },
+  "cache-location": {
+    description:
+      "Path to the cache file or directory\nDefaults to node_modules/.cache/prettier/.prettiercache",
+    type: "path",
```

Comment:
Do we need this?

Reply:
I added `cache-location` options because ESLint has same option. But, I think we can remove this.

Reply:
[af29b16](https://github.com/prettier/prettier/pull/12800/commits/af29b16a788610165f4ebeb839f486e3b5b8c9b8)

Reply:
I think it makes sense to leave `cache-location`. It's a good idea to leave users the option to control where this cache is stored. It's useful in CI at least. Usually, people cache their node_modules by yarn.lock and it could lead to undesired caching of prettier.

Also removing node_modules for reinstall is still a widely spreaded approach and it will wipe all prettier cache

Reply:
Why don't we add it if someone requested?